aspnetcore/host-and-deploy/iis/troubleshoot.md

```diff
@@ -232,11 +232,52 @@ If an app is capable of responding to requests, obtain request, connection, and
 
 ## Slow or hanging app
 
-When an app responds slowly or hangs on a request, obtain and analyze a [dump file](/visualstudio/debugger/using-dump-files). Dump files can be obtained using any of the following tools:
+When an app responds slowly or hangs on a request, obtain and analyze a *crash dump* from [Windows Error Reporting (WER)](/windows/desktop/wer/windows-error-reporting). A crash dump contains a copy of the system's memory at the time of an app crash and can help determine the cause of the crash.
```

Comment:
This only applies when the application actually crashes. To be able to get a dump of an app that's just slow you need to use something different then WER.